rails_programming/introduction/preparing_for_deployment.md

```diff
@@ -0,0 +1,39 @@
+### Introduction
```

Comment:
I don't think we need 2 copies of this file, I think we can use the `seeds` to put this file into both curriculums. I'll have @KevinMulhern take a look

Reply:
The only thing worth pointing out is that the Introduction is slightly different for both.

If it's being consolidated to one file, the initial line beginning in "We have one thing to attend to..." needs to end in a universal manner, as it was modified for the respective courses.

Reply:
@marvingay I think we really need to consolidate these. So that updating one lesson updates both. Especially for the community finding and fixing small issues when the directions get out of date. One may not remember, or know to, update the other.

Reply:
Yeah I think one file for both is the way to go with this as well. For all the same reasons @I3uckwheat stated.